extensions/configuration/certificates.yaml

```diff
@@ -0,0 +1,134 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: certificates.cert.gardener.cloud
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: certificates.cert.gardener.cloud
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |
+    header:
+      - name: status
+        source: 'status.state ? status.state : "UNKNOWN"'
+        highlights:
+          informative:
+          - UNKNOWN
+        widget: Badge
+        description: status.message
+      - name: status.expirationDate
+        source: status.expirationDate
+      - name: commonName
+        source: spec.commonName
+    body:
+      - name: references
+        widget: Panel
+        children:
+          - name: issuer
+            source: status.issuerRef.name
+          - name: secret
+            source: spec.secretRef.name
+          - name: spec.csr
+            source: spec.csr
+          - name: metadata.annotations
+            source: metadata.annotations
+            widget: Labels
+          - name: spec.renew
+            source: spec.renew
+          - name: spec.dnsNames
+            source: spec.dnsNames
+            widget: Labels
+  form: |
+    - var: useCSR
+      dynamicValue: 'false'
+      simple: true
+      type: boolean
+      name: spec.useCSR
+    - simple: true
+      path: spec.commonName
+      required: true
+      placeholder: certificateCN
+      visibility: "$useCSR = false"
+    - simple: true
+      path: spec.csr
+      required: true
+      placeholder: spec.certSignReq
+      visibility: "$useCSR"
+      decodable: true
+      decodedPlaceholder: spec.certSignReqDecoded
```

Comment:
Just as a reminder and information for others and maybe me 😆 , this PR should be merged after this one https://github.com/kyma-project/busola/pull/2288